Source/CNTKv2LibraryDll/API/CNTKLibrary.h

```diff
@@ -2,7 +2,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
 //
-// This is the main header of the CNTK library API containing the entire public API definition. 
+// This is the main header of the CNTK library API containing the entire public API definition.
```

Comment:
My editor did this. Is there any reason for those tailing space/tab in the first place?